context/app/static/js/components/entity-search/searchkit-modifications/withSearchkitRouting.js

```diff
@@ -36,7 +36,6 @@ export const stateToRouteFn = (searchState, fields) => {
     sort: searchState.sortBy,
     filters: searchState.filters,
     fields,
-    size: parseInt(searchState.page?.size),
```

Comment:
Have you been tracking the modifications made to the SK code we've forked? If that's not what you've been doing, probably not worth bothering about it now, but I'd have a slight preference for that.